extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/InjectQuartzSchedulerTest.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import javax.enterprise.event.Observes;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.quartz.Job;
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.SimpleScheduleBuilder;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+
+import io.quarkus.runtime.StartupEvent;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class InjectQuartzSchedulerTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(Starter.class)
+                    .addAsResource(new StringAsset("quarkus.quartz-runtime.force-start=true"),
```

Comment:
```suggestion
                    .addAsResource(new StringAsset("quarkus.quartz.force-start=true"),
```

Reply:
Unfortunately, `quarkus.quartz.force-start=true` would not work. See https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/.60RuntimeConfig.60.20vs.20.60RunTimeConfig.60

Reply:
I'm going to send a PR to fix https://github.com/quarkusio/quarkus/issues/7257 - afterwards, it should be possible to switch to `quarkus.quartz.force-start=true`.

Reply:
Thanks @mkouba